msgpack/_packer.pyx

```diff
@@ -174,10 +184,11 @@ cdef class Packer(object):
                 else:
                    dval = o
                    ret = msgpack_pack_double(&self.pk, dval)
-            elif PyBytes_CheckExact(o) if strict_types else PyBytes_Check(o):
+            elif PyBytesLike_CheckExact(o) if strict_types else PyBytesLike_Check(o):
                 L = len(o)
                 if L > ITEM_LIMIT:
-                    raise PackValueError("bytes is too large")
+                    raise PackValueError("%s is too large"
+                                         % type(o).__name__.capitalize())
```

Comment:
I don't like `capitalize()`.
How about `"%s length is too large" % type(o).__name__`?